src/Service/FormService.php

```diff
@@ -58,13 +58,13 @@ public function __construct(
         $this->liform = $liform;
     }
 
-    public function build(string $name): FormBuilderInterface
+    public function build(string $name, mixed $context = null): FormBuilderInterface
```

Comment:
Extend the following functions with `context`    
     /**
     * @return array<mixed>
     */
    public function buildJson(string $name): array
    {
        return $this->json($this->buildForm($name));
    }

    public function buildJsonString(string $name): string
    {
        return json_encode($this->buildJson($name), \JSON_THROW_ON_ERROR);
    }

    public function buildForm(string $name): FormInterface
    {
        return $this->build($name)->getForm();
    }